### DISCUSSION TITLE
colorparsley's AGPL license forces us to stop using apca-w3

### DISCUSSION THREAD

**Discussion by Ariane-B** [Myndex/apca-w3#11] [General]
We were using apca-w3 to determine the appropriate foreground color for a given background color. It works great.

But we have to stop using it immediately, because we discovered colorparsley has an AGPL licence, which would force us to open source our whole app, which our business model and clients wouldn't allow.

We're likely not the only closed-source app that would like to use apca-w3.

I thought you might want to know. If being off-limits to any closed-source project sounds bad to you, you might want to reconsider the dependency.

**Comment by Myndex** [comment]
Hi @Ariane-B 

Thank you very much for your comment, that was certainly not the intention as we created and are providing ColorParsley and other dependencies.

To alleviate your concern we have added the following line to the license to clarify:


## DEPENDENCIES
Any dependencies created by and provided by Myndex Research or Andrew Somers as part of the apca-w3 package @ npm, or this repository, are hereby included by reference within this license, including but not limited to the Color Parsley package, when these dependencies are used with APCA – W3.

-----

Please let me know if you have any further questions, and sorry about any concerns this might've raised. And to be clear, APCA – W3 does not require Color Parsley it is added as a convenience.

Thank you for reading,

Andy
